src/host/main.cpp

```diff
@@ -102,6 +104,11 @@ void setup() {
 
   BatteryMonitorPage *batPage = new BatteryMonitorPage(skHub);
   mfd.addPage(batPage);
+	
+	#ifdef IMU_MONITOR_PAGE
+    IMUMonitorPage *imuPage = new IMUMonitorPage(skHub);
```

Comment:
To allow `IMUMonitorPage` to have more access to `IMUService`, pass a reference to the `IMUService` when you create `IMUMonitorPage`: 

```
IMUMonitorPage *imuPage = new IMUMonitorPage(skHub, *imuService);
```

And then in `IMUService.h` you can declare new functions for example:
```
public:
  void getLastValues(int &accelCalibration, double &pitch, double &heel, int &magCalibration, double &heading);
  void saveAccelCalibration();
  void resetAccelCalibration();
  void saveMagCalibration();
  void resetMagCalibration();
```
(and anything else you need)

This way, the `imuPage` will have more access to imuService but we will keep the imu logic inside the imu service.

Reply:
This is looks nice, but I do not know how to declare 
```IMUMonitorPage(SKHub& hub, IMUService* ims);```
this gives me an error "redefinition of class IMUService"
how to do this right?